22-discovery.ks

```diff
@@ -98,17 +98,12 @@ systemctl set-default multi-user.target
 echo " * setting up journald and ttys"
 systemctl disable getty@tty1.service getty@tty2.service
 systemctl mask getty@tty1.service getty@tty2.service
-echo "SystemMaxUse=15M" >> /etc/systemd/journald.conf
+echo "Storage=volatile" >> /etc/systemd/journald.conf
+echo "RuntimeMaxUse=15M" >> /etc/systemd/journald.conf
```

Comment:
Rebased and amended one more change her. The `volatile` option prevents journald from creating `/var/lib/journal`, it only uses in-memory database of maximum size 15 MBs. Saves a bit of memory because the empty persistent journal is about 30 MB in size.